webpack/assets/javascripts/react_app/components/charts/StatisticsChartsListStyles.js

```diff
@@ -0,0 +1,27 @@
+export default {
```

Comment:
React inline styles is sass on steroids.

Although the final artifact is the same, implementing css in js differs from writing styles directly in html tags. Using composable javascript objects provides a powerful, easy to use and less error prone tool for managing components styles.

While we are using a combination of css and inline styles it is also possible to implement global styles with the inline styles paradigm.

It is possible to build React components that allow style overriding. Have a look at the MessageBox component.

Dependencies are more explicit.